### PR TITLE
fixing commands, adding note, other edits

### PR DESCRIPTION
**Pull request by anaxceron**
Describe your pull request here: Updates to address [Issue 3440](https://github.com/zowe/docs-site/issues/3440)

List the file(s) included in this PR: cli-using-integrating-apiml.md

After creating the PR, follow the instructions in the comments.

### CODE REVIEW

**Review by t1m0thyj — APPROVED** [review]
LGTM, thanks @anaxceron!

**Review by adam-wolfe — APPROVED** [review]
LGTM

File: docs/appendix/zowe-glossary.md [hunk 1]
```diff
@@ -185,9 +191,14 @@ The governing body that is responsible for the overall planning, development, an
 
 The Zowe Support Provider Conformance Program gives vendors the ability to showcase their Zowe support competencies via well defined criteria. It is administered by the Linux Foundation and Open Mainframe Project.
```
Comment (sbohnert) [inline docs/appendix/zowe-glossary.md:192]: Avoid the use of the word "via" and other similar expressions

File: docs/appendix/zowe-glossary.md [hunk 2]
```diff
@@ -212,6 +223,10 @@ Use of z/OS UNIX services requires a z/OS UNIX security context, referred to as
 
 The z/OS Unix directory for the [Zowe runtime](#zowe-runtime), specified in the Zowe configuration file via `zowe.runtimeDirectory`. Also the parent directory of the `zwe` command.
 
+#### Service profile
+
+A type of team configuration profile that stores connection information for a specific mainframe service, such as IBM z/OSMF. Plug-ins can introduce other service profile types, such as the CICS profile to connect to IBM CICS.
+
 #### SMP/E
 
 The Zowe installation for Zowe z/OS components that is distributed as an SMP/E package, identified by FMID, and contains the runtimes and the scripts to install and launch the z/OS runtime. The initial package is installed and then a PTF is applied. It is the second most common method to install Zowe.
```
Comment (sbohnert) [inline docs/appendix/zowe-glossary.md:232]: Missing a comma:
The initial package is installed, and then a PTF is applied.

File: docs/user-guide/cli-using-integrating-apiml.md [hunk 3]
```diff
@@ -1,14 +1,18 @@
...
 
-Zowe API Mediation Layer (ML) provides a single point of access to a defined set of mainframe services. The layer provides API management features such as high-availability, consistent security, and a single sign-on (SSO) and multi-factor authentication (MFA) experience.
+Zowe API Mediation Layer (ML) provides a secure single point of access to a defined set of mainframe services. The layer provides API management features such as high-availability, consistent security, and a single sign-on (SSO) and multi-factor authentication (MFA) experience.
 
-You can access services through API ML without reauthenticating every time you issue a command. Tokens allow for a secure interaction between the client and server. When you issue commands to API ML, the layer routes requests to an appropriate API instance based on system load and available API instances.
+You use tokens to access services through API ML without reauthenticating every time you issue a command. Tokens allow for secure interaction between the client and server. When you issue commands to API ML, the layer routes requests to an appropriate API instance based on system load and available API instances.
```
Comment (sbohnert) [inline docs/user-guide/cli-using-integrating-apiml.md:5]: Change
You use tokens to access services through API ML without reauthenticating every time you issue a command.
to 
Use tokens to access services through API ML without reauthenticating every time you issue a command.

**Review by sbohnert — COMMENTED** [review]
Looks good. Thanks

**Pull request merged**